crates/edr_solidity/src/opcodes.rs

```diff
@@ -0,0 +1,17 @@
+use revm::interpreter::opcode;
+
+fn is_push(opcode: u8) -> bool {
+    (opcode::PUSH1..=opcode::PUSH32).contains(&opcode)
+}
+
+fn get_push_length(opcode: u8) -> usize {
```

Comment:
Sorry, I keep doing this. I fixed the ones you mentioned in 040048592ac85d46898978f8298e88370cdadb30. There are still two more: `get_max_match` and `get_shared_prefix_length`. Should those be renamed too, or are those fine because they are not "straightforward" getters?

Reply:
For getters there is no requirement that they return instantly. If you want to be explicit about there being different APIs, one for return a cached value vs another calculating the value; you can use another word compared to `get`, such as `calculate_` or `determine_`